run/archiveloop

```diff
@@ -246,18 +246,22 @@ function disconnect_usb_drives_from_host () {
 
 function archive_teslacam_clips () {
   log "Starting recording archiving..."
+  
+  /root/bin/send-push-message "TeslaUSB: DashCam Archive started" "Dashcam archiving started at $(date)"
 
   ensure_cam_file_is_mounted
 
   fix_errors_in_cam_file
-
+  
   /root/bin/archive-clips.sh
 
   unmount_cam_file
 }
 
 function copy_music_files () {
   log "Starting music sync..."
+  
+  /root/bin/send-push-message "TeslaUSB: Music copy started" "Music copy started at $(date)"
```

Comment:
I could see this getting annoying pretty quickly, since it will be sent even if it ends up not copying anything because there was nothing new to copy.